test_receiver.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
```

Comment:
If this is meant to be a test, can you write this in a way that it will actually pass / fail on travis? Aka, make it a unit test instead of printing things and requiring manual work to verify that it is printing the right stuff.